rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/RascalLanguageServices.java

```diff
@@ -28,7 +28,6 @@
 
 import static org.rascalmpl.vscode.lsp.util.EvaluatorUtil.makeFutureEvaluator;
 import static org.rascalmpl.vscode.lsp.util.EvaluatorUtil.runEvaluator;
-
```

Comment:
Did those disappear due to automatic import optimization? I think we should reduce changes in order/layout as much as possible here. Perhaps we align our editor configs or sync settings.

Reply:
This happens due to optimize imports that I have in my muscle memory to drop unused imports.

Reply:
Understood. But I also use optimize imports, and mine does not drop empty lines, for example. I think we should make sure to align settings as much as possible to prevent messy diffs.

Reply:
I assumed it was a result of some kind of merge conflict, or someone manually added a newline. I have no specific config, except that I've installed the editor config plugin. That helps normalize these kinds of things on a per repo basis.